Configuration/TCA/tx_styleguide_type.php

```diff
@@ -88,18 +88,32 @@
             ]
         ],
 
-        'type' => [
-            'exclude' => 1,
+        'record_type' => [
             'label' => 'type',
             'config' => [
                 'type' => 'select',
                 'renderType' => 'selectSingle',
                 'items' => [
                     ['type 0', '0'],
+                    ['Type with changed fields', 'withChangedFields'],
+                    ['Type with columnsOverrides', 'withOverriddenColumns'],
                     ['type test', 'test'],
                 ],
             ],
         ],
+        'title' => [
```

Comment:
i hope it's ok if i change these field names to the casual naming that styleguide uses everywhere else, too.